qmake/compiler.pri

```diff
@@ -7,7 +7,36 @@ include(qt.pri)
 include(uname.pri)
 include(buildenv.pri)
 include(cplusplus.pri)
- 
+
+# Globally use "/debug" or "/release"
```

Comment:
Maybe move this to qmake/builddir.pri, and include it at the top of compiler.pri?

Reply:
Yes.